ReactWindows/ReactNative.Shared/UIManager/UIManagerModule.cs

```diff
@@ -511,6 +516,39 @@ public void configureNextLayoutAnimation(JObject config, ICallback success, ICal
 
         #endregion
 
+        #region Sync React Methods
+
+        /// <summary>
+        /// Gets the constants for the view manager.
+        /// </summary>
+        /// <param name="viewManagerName">The view manager name.</param>
+        /// <returns>The view manager constants.</returns>
+        [ReactMethod(IsBlockingSynchronousMethod = true)]
+        public IDictionary<string, object> getConstantsForViewManager(string viewManagerName)
+        {
+            var viewManager = _uiImplementation.ResolveViewManager(viewManagerName);
+
+            using (Tracer.Trace(Tracer.TRACE_TAG_REACT_BRIDGE, "UIManagerModule.getConstantsForViewManager")
+                .With("ViewManager", viewManagerName)
+                .With("Lazy", true)
+                .Start())
+            {
+                return CreateConstantsForViewManager(viewManager, null, null, null, _customDirectEvents);
+            }
+        }
+
+        /// <summary>
+        /// Gets the default event types for the view manager.
```

Comment:
>the view manager [](start = 45, length = 16)

This is not per view manager, the comment is misleading

Reply:
Yeah, I think I meant UIManager, good catch.

---
In reply to: [174849563](https://github.com/Microsoft/react-native-windows/pull/1717#discussion_r174849563) [](ancestors = 174849563)